feature-utils/poly-look/src/react-components/overlays/sideSheet.css

```diff
@@ -15,16 +15,19 @@
   display: flex;
   align-items: center;
   min-height: 48px;
-}
-.poly-side-sheet .close-button-left {
-  position: absolute;
-  left: 0;
+  position: relative;
+  margin: 0 -16px;
```

Comment:
This is doing two things. One, it I guess it fixes what it says, the "potential title break". But second, and apparently unrelated, is to eliminate specific classes for `close-button-left`. However, that has not been eliminated, it's still here:

https://github.com/polypoly-eu/polyPod/pull/853/files#diff-89aa112ea459c030097834f4816b8aa19fdcd11735ce87ce3fe8b4e24405553aL30

I haven't seen it somewhere else, is there something I'm missing here?

Reply:
Thank you! I've missed that. Will fix with the next commit. The context around that commit was: "Ok, I'm done done, I can create the PR. I wonder how the german version of the title looks like, gonna try it out and then PR.." And then I see the title going under the button.... 💔 So I've rushed a bit, but still did my best to test it out. The reason behind that 48px padding is that the button is 48px so no matter how little space there is, the title still stays centered and it doesn't go under/over the button.
I don't consider CSS one of my strengths, so if there is an elegant way of doing this, I would really like to know.

Reply:
I don't think anyone lists in their resumé "Strong in CSS" :rofl: